packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewer.stories.js

```diff
@@ -16,9 +17,8 @@ const config = {
 }
 
 const subject = Factory.build('subject', {
-  locations: [
-    { 'text/plain': 'https://panoptes-uploads-staging.zooniverse.org/subject_location/9cd82380-5c44-40ef-a0da-dcafb01195e1.txt' }
-  ]
+  content: 'Herbarium of the University of North Carolina\nSOUTH CAROLINA\nCharleston County\nGnaphalium peregrinum Fern,\nrailroad right-of-way, Johns Island Station on\nCounty Rt. 20 (wes t of Charleston.\nHarry E. Ahles 22002 April 2, 1957\nwith John G. Haesloop\nCollected for the “Flora of the Carolinas"',
```

Comment:
The text is illegible in dark mode.
<img width="785" alt="Screenshot of the test subject rendered as black text on a dark grey background." src="https://user-images.githubusercontent.com/59547/159326287-b7690520-db15-47aa-a49e-5d125a5fac9b.png">

Reply:
The viewer in app-project or lib-classifier reflects the zooGrommet theme text colors for light and dark, which are legible, but I could not figure out how to have the story match (respect the theme) in a straightforward way. I've pushed some changes that brute force set the styling to match light and dark modes with a Grommet Text component, which is not ideal, but I'm not sure how else to fix.

Reply:
Me neither. I'd have thought the default text colour would be set by the theme provider.